package.json

```
@@ -123,7 +123,9 @@
     "typescript": "^3.9.0",
     "ua-parser-js": "^0.7.28",
     "url-parse": "^1.5.0",
-    "xmlhttprequest-ssl": "^1.6.2"
+    "xmlhttprequest-ssl": "^1.6.2",
+    "dns-packet": "5.2.2",
```

This can be changed to `^1.3.4` since the change was backported to the v1 branch: https://github.com/mafintosh/multicast-dns/issues/75#issuecomment-848560322

```suggestion
    "dns-packet": "^1.3.4",
```